app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -39,6 +41,9 @@ class MainActivity : AppCompatActivity() {
             this,
             R.id.nav_host_fragment
         )
+        /**
+         * Sets up the navigation bar on the bottom of the screen.
+         */
```

Comment:
Added annotation and javadoc.

Reply:
It's not common to place JavaDoc inside of a method.  Inline comments go inside.  
I agree with what you say here: "the annotation itself should be self describing and obvious as meta-data, so there should only little documentation itself."
so I don't know that you even need documentation here... this comment pretty much repeats the method name, which is self describing.